### ISSUE TITLE
BrawlBox Audio Problem

### ISSUE THREAD

**Issue by Mogu94** [libertyernie/brawltools#187]
When opening up a WAV file to loop and make a BRSTM out of, the program is very buggy. I pause at a specific place and click to make that the starting point for the loop. Then I press play to make sure I got the starting point exactly where I want it and it turns out the Audio is being delayed or something. It'll start playing the last second of sound before the place I was making the starting point for the loop.

Basically, if I play and then pause the song at a point and then hit play again, the audio plays the previous second of sound before.

**Comment by Mogu94** [comment]
https://youtu.be/Xqc36gD_YM8 Here's a video

**Comment by libertyernie** [comment]
Can you post the wav file itself? Maybe that will help me reproduce the issue.

**Comment by gheskett** [comment]
I'm not able to reproduce this bug either, but I can't help but notice the time display in that video.  It's definitely incrementing more slowly than it should be.  I don't know how that display is updated, but it might be something to take a look at.

**Comment by libertyernie** [comment]
@MDHEXT Could you create a separate issue for this? Also, conversion from wav to brstm or brstm to wav?
